hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -159,7 +159,7 @@ checkCollectCom ctx@ScriptContext{scriptContextTxInfo = txInfo} (contestationPer
 
   mustInitVersion =
     traceIfFalse $(errorCode IncorrectVersion) $
-      version' == sn && sn == 0
```

Comment:
i know its redundant, but why?

Reply:
There is no snapshot number in the open state anymore